### PR TITLE
Created Transport.java

### PR DESCRIPTION
**Pull request by vlad-terebylo**

### CODE REVIEW

File: Project8/src/TransportSystem/Airplane.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package TransportSystem;
+
+public class Airplane implements Transport{
+    private int destination;
+
+    public Airplane() {
```
Comment (scratchw0w) [inline Project8/src/TransportSystem/Airplane.java]: do we really need default constructor here?

please remove it if not

File: Project8/src/TransportSystem/Airplane.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    }
+    public Airplane(int destination){
+        this.destination = destination;
+    }
+    public void move(){
```
Comment (scratchw0w) [inline Project8/src/TransportSystem/Airplane.java]: add enter between functions

File: Project8/src/TransportSystem/Bicycle.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package TransportSystem;
+
+public class Bicycle implements Transport {
+    private int destination;
+
+    public Bicycle() {
```
Comment (scratchw0w) [inline Project8/src/TransportSystem/Bicycle.java]: and here

File: Project8/src/TransportSystem/Car.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package TransportSystem;
+
+public class Car implements Transport{
+    private int destination;
+
+    public Car(){}
```
Comment (scratchw0w) [inline Project8/src/TransportSystem/Car.java]: and here

File: Project8/src/TransportSystem/Airplane.java (hunk 2 above)
Comment (scratchw0w) [inline Project8/src/TransportSystem/Airplane.java]: add @Override annotation

File: Project8/src/TransportSystem/Bicycle.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    }
+
+    public Bicycle(int destination) {
+        this.destination = destination;
+    }
+
+    public void move() {
```
Comment (scratchw0w) [inline Project8/src/TransportSystem/Bicycle.java:12]: add @Override annotation

File: Project8/src/TransportSystem/Transport.java [hunk 6]
```diff
@@ -0,0 +1,5 @@
+package TransportSystem;
+
+public interface Transport {
+    void move();
```
Comment (scratchw0w) [inline Project8/src/TransportSystem/Transport.java:4]: make move not only print current distance, but to move distance

**Review by scratchw0w — CHANGES_REQUESTED** [review]
Please fix comments

File: Project8/src/TransportSystem/Airplane.java (hunk 1 above)
Comment (vlad-terebylo) [reply Project8/src/TransportSystem/Airplane.java]: Fixed

File: Project8/src/TransportSystem/Airplane.java (hunk 2 above)
Comment (vlad-terebylo) [reply Project8/src/TransportSystem/Airplane.java]: Fixed

File: Project8/src/TransportSystem/Bicycle.java (hunk 3 above)
Comment (vlad-terebylo) [reply Project8/src/TransportSystem/Bicycle.java]: Fixed

File: Project8/src/TransportSystem/Car.java (hunk 4 above)
Comment (vlad-terebylo) [reply Project8/src/TransportSystem/Car.java]: Fixed

File: Project8/src/TransportSystem/Airplane.java (hunk 2 above)
Comment (vlad-terebylo) [reply Project8/src/TransportSystem/Airplane.java]: Fixed

File: Project8/src/TransportSystem/Bicycle.java (hunk 5 above)
Comment (vlad-terebylo) [reply Project8/src/TransportSystem/Bicycle.java:12]: Fixed

File: Project8/src/TransportSystem/Transport.java (hunk 6 above)
Comment (vlad-terebylo) [reply Project8/src/TransportSystem/Transport.java:4]: Fixed

**Review by scratchw0w — APPROVED** [review]
Good job!

**Pull request merged**